File: _01_Java_Basics/Fizzbuzz.java
package _01_Java_Basics;

public class Fizzbuzz {
    public static void calculate() {
        for (int i = 0; i < 100; i++) {
            if (i % 3 == 0 && i % 5 == 0)
                System.out.println("FizzBuzz");
            else if (i % 3 == 0)
                System.out.println("Fizz");
            else if (i % 5 == 0)
                System.out.println("Buzz");
        }
    }
    public static void main(String[] args) {
        Fizzbuzz.calculate();
    }
    
}

File: _01_Java_Basics/Fizzbuzz.py
def Fizzbuzz(n):
    for i in range(n):
        if (i % 15 == 0):
            print("FizzBuzz")
        elif (i % 3 == 0):
            print("Fizz")
        elif (i % 5 == 0):
            print("Buzz")


number = int(input("Enter number: "))
Fizzbuzz(number)
            